refinery/core/api.py

```diff
@@ -498,6 +498,13 @@ class Meta:
         }
 
     def dehydrate(self, bundle):
+        version_details = bundle.obj.get_version_details()
```

Comment:
`get_version_details()` is not named correctly because it actually returns an InvestigationLink instance, so I would rename the variable holding the return value to something like `investigation_link`